pkg/virtual/framework/forwardingregistry/rest_test.go

```diff
@@ -59,9 +59,10 @@ func (mcg *mockedClusterClient) Cluster(cluster logicalcluster.Name) dynamic.Int
 	return mcg.client
 }
 
-func newStorage(t *testing.T, clusterClient dynamic.ClusterInterface, patchConflictRetryBackoff *wait.Backoff) customresource.CustomResourceStorage {
-	groupResource := schema.GroupResource{Group: "mygroup.example.com", Resource: "noxus"}
-	gvr := groupResource.WithVersion("v1beta1")
+var noxusGVR schema.GroupVersionResource = schema.GroupVersionResource{Group: "mygroup.example.com", Resource: "noxus", Version: "v1beta1"}
```

Comment:
noxus is back 😱

Reply:
was already there :-)
Copied this from somewhere initially. No idea where it comes from.

Reply:
I think it's a deads2k special 😄